packages/block-library/src/list-item/edit.native.js

```diff
@@ -85,21 +85,32 @@ export default function ListItemEdit( {
 	const blockProps = useBlockProps( {
 		...( hasInnerBlocks && styles[ 'wp-block-list-item__nested-blocks' ] ),
 	} );
+
 	const innerBlocksProps = useInnerBlocksProps( blockProps, {
 		allowedBlocks: [ 'core/list' ],
 		renderAppender: false,
 	} );
 
-	const placeholderWithPreferredScheme = usePreferredColorSchemeStyle(
+	// Set default placeholder text color from light/dark scheme or base colors
+	const defaultPlaceholderFromScheme = usePreferredColorSchemeStyle(
 		styles[ 'wp-block-list-item__list-item-placeholder' ],
 		styles[ 'wp-block-list-item__list-item-placeholder--dark' ]
 	);
 
-	// Add 0.62 hex opacity to baseColor text (e.g. #FFFFFF9E)
-	// or fallback to default color from stylesheet ($gray, $gray-50)
 	const defaultPlaceholderTextColor = style?.baseColors?.color?.text
-		? style.baseColors.color.text + '9E'
-		: placeholderWithPreferredScheme?.color;
+		? styles.color || style.baseColors.color.text
+		: defaultPlaceholderFromScheme?.color;
+
+	// Add hex opacity to default placeholder text color and style object
+	const defaultPlaceholderTextColorWithOpacity =
+		defaultPlaceholderTextColor + '9e';
+
+	const styleWithPlaceholderOpacity = {
+		...style,
+		...( style.color && {
+			placeholderColor: style.color + '9e',
```

Comment:
This looks good to me but now I'm thinking if we should just add this opacity to all placeholders that get set from the global styles data. 🤔

What do you think if we just add `9e` [here](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/mobile/global-styles-context/utils.native.js#L131)?

Reply:
@geriux Yeah, I agree that the placeholder color would be better be set somewhere more globally. I went back and forth on a few different solutions here, because the placeholder color value can come from some many different places. 

I think this List block code is the only place where we are using opacity to handle the color change. The paragraph block placeholder is the same color as the text input, for example. But perhaps the paragraph block does it this way  purpose, by design? 

I wondering if you think we should try to add `9e` to all hex values globally, or start with the list block implementation here and get more design feedback?

Reply:
> But perhaps the paragraph block does it this way purpose, by design?

I think we missed tweaking the placeholder color for the Paragraph block when we added support for block-based themes, but it's not much as an issue as the List I think.

> I wondering if you think we should try to add 9e to all hex values globally, or start with the list block implementation here and get more design feedback?

Let's keep it as it is now for the List block and see if it makes sense to move it to the global styles context util. 👍